src/swift/Block.swift

```diff
@@ -104,8 +104,8 @@ public class DispatchWorkItem {
 /// uses @convention(block). This is to avoid unnecessary bridging between
 /// C blocks and Swift closures, which interferes with dispatch APIs that depend
 /// on the referential identity of a block. Particularly, dispatch_block_create.
-internal typealias _DispatchBlock = @convention(block) () -> Void
-internal typealias dispatch_block_t = @convention(block) () -> Void
+internal typealias _DispatchBlock = @escaping @convention(block) () -> Void
```

Comment:
This will fail to compile on swift-3.0-branch as of yesterday. You cannot put @escaping anywhere except for a function parameter type.

Basically, if your parameter is written as just `_DispatchBlock`, it will be non-escaping. If you write `@escaping _DispatchBlock`, it is escaping.